torch/_dynamo/variables/builder.py

```diff
@@ -1228,6 +1233,8 @@ def _clone_input(value):
             return TupleVariable(unpacked, **options)
         elif istype(example_value, (list, immutable_list)):
             return ListVariable(unpacked, mutable_local=MutableLocal(), **options)
+        elif istype(example_value, set):
```

Comment:
How much more difficult would it be to support "frozenset" as well?

Reply:
Frozenset creation as a local, probably pretty easy. As an input? No go for the same reason as set.